pkg/glib/data.go

```diff
@@ -0,0 +1,127 @@
+// Package glib contains methods for interactions with GSuite API
+// This file contains static-data for the methods
```

Comment:
The comment block immediately preceding a package declaration is the package-level documentation. I'm not sure what happens if two files in the same package have different comments for the package declaration, which is why these comments are usually only done in one file, often called * doc.go*  and left empty except for the comment and declaration.

This also means that any "This file..." remarks in this comment will be misleading, as from looking at the documentation only, the reader would have no idea what file is meant by "this file". Better to instead put a comment on the `var googleLicenses` declaration.